commit 8bdc9fe91da35790f4238cf0837e4e44933a6eeb (refs/changes/22/5722/3)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2019-03-06T16:55:48+00:00 (7 months ago)
    
    OS-7636 ltp instructions need to clarify ltp version to use
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/lib/brand/lx/testing/Readme_ltp b/usr/src/lib/brand/lx/testing/Readme_ltp
index 4a2fcd353b..5165546c1a 100644
--- a/usr/src/lib/brand/lx/testing/Readme_ltp
+++ b/usr/src/lib/brand/lx/testing/Readme_ltp
@@ -19,6 +19,7 @@ Additional prerequisites are required activate some tests:
 As a normal user:
     git clone https://github.com/linux-test-project/ltp.git
     cd ltp
+    git checkout ed01f6a05c77f65cb5d1089474c9a0e2129c581a
     make autotools
     ./configure
     make all
@@ -50,7 +51,7 @@ which now work.
 As root:
     cd /opt/ltp
     /opt/ltp/runltp -f `cat /native/usr/lib/brand/lx/ltp_tests` \
-        -S /native/usr/lib/brand/lx/ltp_skiplist -p >/tmp/test.log
+        -S /native/usr/lib/brand/lx/ltp_skiplist -p >/tmp/test.log 2>&1
 
 When the test run has finished, the results will be logged in a date/time
 stamped file under /opt/ltp/results. The summary at the end of the log file
